assets/js/formbuilder.js

```
@@ -0,0 +1,840 @@
+                    if (attrPhp) line += ', ' + attrPhp;
+                    if (item.defaultValue) line += ', null, ' + phpStr(item.defaultValue);
+                    break;
+                case 'hidden':
+                    line += ', ' + (item.defaultValue ? phpStr(item.defaultValue) : 'null');
+                    break;
+                case 'media':
+                case 'medialist':
+                case 'imagelist':
+                case 'link':
+                case 'linklist':
+                    var catLit = item.category ? (parseInt(item.category, 10) || phpStr(item.category)) : 'null';
```

Biome (2.4.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)